hadoop-hdds/interface-server/src/main/proto/ScmServerDatanodeHeartbeatProtocol.proto

```diff
@@ -337,6 +337,10 @@ message DeletedBlocksTransaction {
   required int32 count = 4;
 }
 
+message DeletedBlocksTransactionIDs {
+  repeated int64 txID = 1;
```

Comment:
what would be the upgrade implications here after this proto change?

Reply:
What does `upgrade implications` means ? could you please explain more ?